src/TunePID.cpp

```diff
@@ -76,8 +77,11 @@ bool initMotor(motorid_t motor) {
 		robot::encparams_t enc_params = robot::encMotors.at(motor);
 		can::motor::initEncoder(serial, enc_params.isInverted, true, enc_params.ppjr,
 								robot::TELEM_PERIOD);
-		can::motor::setLimitSwitchLimits(serial, enc_params.limitSwitchLow,
-										 enc_params.limitSwitchHigh);
+		auto mDegToPulses = [&](int32_t mdeg) {
+			return static_cast<int32_t>((double)mdeg / 360000.0 * enc_params.ppjr);
+		};
+		can::motor::setLimitSwitchLimits(serial, mDegToPulses(enc_params.limitSwitchLow),
+										 mDegToPulses(enc_params.limitSwitchHigh));
```

Comment:
The parameters to `setLimitSwitchLimits()` should be in encoder ticks, not mdegs. This used to work before, only because it was incorrect on both sides (firmware and software) but disagreed with documentation. This has been updated on both sides.